Source/Core/ServiceTask+ObjC.swift

```diff
@@ -132,9 +132,7 @@ extension ServiceTask {
     */
     @objc public func updateUIObjC(handler: (AnyObject?) -> Void) -> Self {
         return updateUI { value in
-            if let value = value as? AnyObject {
-                handler(value)
-            }
+            handler(value as! AnyObject?)
         }
```

Comment:
Originally I did have an else in there with a fatalError() but it seemed unnecessary since it's just raising the same error that the runtime would throw.

```
if let value = value as? AnyObject? {
    handler(value)
} else {
    fatalError("Failed to downcast Any? type to AnyObject?")
}
```